src/Rings/groebner.jl

```diff
@@ -1221,3 +1248,126 @@ function _compute_groebner_basis_using_fglm(I::MPolyIdeal,
 	dim(I) == 0 || error("Dimension of ideal must be zero.")
 	I.gb[destination_ordering] = _fglm(G, destination_ordering)
 end
+
+
+@doc Markdown.doc"""
+    groebner_basis_hilbert_driven(I::MPolyIdeal,
+                                  destination_ordering::MonomialOrdering;
```

Comment:
Please rename it to `ordering` and make it a keyword argument to coincide with our naming/argument convention.

Reply:
What should be the default value for `ordering`?

Reply:
I would propose that we do not have default value here. For me it is the same as with `fglm`, we do not know the destination ordering of the GB, this is more of a specialized function where we expect the users to know what they want.